plugins/performance-lab/tests/test-load.php

```diff
@@ -7,6 +7,17 @@
 
 class Test_Load extends WP_UnitTestCase {
 
+	/**
+	 * Runs the routine before each test is executed.
+	 */
+	public function set_up(): void {
+		parent::set_up();
+
+		if ( ! defined( 'PERFLAB_PLACE_OBJECT_CACHE_DROPIN' ) ) {
+			define( 'PERFLAB_PLACE_OBJECT_CACHE_DROPIN', true );
+		}
```

Comment:
Maybe add a comment to clarify why this is here:
```suggestion
		/*
		 * This constant is not set by default in production.
		 * However, it is needed for all tests that cover placement of the object cache drop-in.
		 */
		if ( ! defined( 'PERFLAB_PLACE_OBJECT_CACHE_DROPIN' ) ) {
			define( 'PERFLAB_PLACE_OBJECT_CACHE_DROPIN', true );
		}
```